### PR TITLE
Remove unnecessary code

### PR DESCRIPTION
**Pull request by sarathrasher**
Removed lines to check for errors, including console.logs and an error function on my getData function, because the code is running with no errors. 

@nat-sharpe

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -61,7 +62,7 @@ var deleteOrder = function (order) {
         success: function(orderObject) {
             console.log(order);
         }
-});
+    });
```
Comment (nat-sharpe) [inline index.js]: Looks good

File: index.js [hunk 2]
```diff
@@ -21,7 +21,8 @@ var printOrder = function (order) {
     orderList.appendChild(orderListItem);
 
     var removeOrder = function (event) {
-        orderList.removeChild(orderListItem)
+        orderListItem.classList.add('green');
```
Comment (nat-sharpe) [inline index.js:24]: WHAT IS GOING ON????

**Comment by nat-sharpe** [comment]
👍

File: index.js (hunk 2 above)
Comment (sarathrasher) [reply index.js:24]: Added new feature: moved the remove functionality inside a timeOut function so it operated on a delay

**Pull request merged**